server/server.go

```diff
@@ -0,0 +1,123 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+
+	"github.com/Clever/cron-admin/db"
+)
+
+func byteHandler(handler func(*http.Request) ([]byte, error)) func(http.ResponseWriter, *http.Request) {
+	return func(rw http.ResponseWriter, req *http.Request) {
+		data, err := handler(req)
+		if err != nil {
+			fmt.Printf("Received handler err: %s\n", err)
+		}
+		rw.Write(data)
+	}
+}
+
+func jsonHandler(handler func(*http.Request) (interface{}, error)) func(http.ResponseWriter, *http.Request) {
+	return func(rw http.ResponseWriter, req *http.Request) {
+		wrappedHandler := func(req *http.Request) ([]byte, error) {
+			resp, err := handler(req)
+			if err != nil {
+				return nil, err
+			}
+			data, err := json.Marshal(resp)
+			if err != nil {
+				return nil, err
+			}
+			rw.Header().Add("Content-Type", "application/json; charset=utf-8")
+			return data, nil
+		}
+		byteHandler(wrappedHandler)(rw, req)
+	}
+}
+
+func SetupHandlers(r *mux.Router, db *mongodb.DB) {
+	r.HandleFunc("/active-jobs", jsonHandler(func(req *http.Request) (interface{}, error) {
```

Comment:
consider changing this to a `/jobs` endpoint with an optional `?active=true` filter